packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -41,6 +41,7 @@ class ExpansionTile extends StatefulWidget {
     this.children = const <Widget>[],
     this.trailing,
     this.initiallyExpanded = false,
+    this.padding,
```

Comment:
I would name this property `tilePadding` or something better (I can't think of anything more descriptive so far). I think if we call this `padding`, then it sounds like it insets the entire widget, including when the tile expands its children.

Reply:
`tilePadding` seems apt.